src/UI/Badge.elm

```diff
@@ -34,54 +36,62 @@ type BadgeTone
 
 light : String -> Badge
 light content =
-    Badge { content = content } { tone = ToneLight }
+    Badge { content = content } { defaultOptions | tone = ToneLight }
 
 
 dark : String -> Badge
 dark content =
-    Badge { content = content } { tone = ToneDark }
+    Badge { content = content } { defaultOptions | tone = ToneDark }
 
 
 primary : String -> Badge
 primary content =
-    Badge { content = content } { tone = TonePrimary }
+    Badge { content = content } { defaultOptions | tone = TonePrimary }
 
 
 warning : String -> Badge
 warning content =
-    Badge { content = content } { tone = ToneWarning }
+    Badge { content = content } { defaultOptions | tone = ToneWarning }
 
 
 danger : String -> Badge
 danger content =
-    Badge { content = content } { tone = ToneDanger }
+    Badge { content = content } { defaultOptions | tone = ToneDanger }
 
 
 success : String -> Badge
 success content =
-    Badge { content = content } { tone = ToneSuccess }
+    Badge { content = content } { defaultOptions | tone = ToneSuccess }
+
+
+withAlpha : Float -> Badge -> Badge
```

Comment:
![Screenshot 2020-05-22 at 12 56 24](https://user-images.githubusercontent.com/1559013/82661127-b60c2f00-9c2b-11ea-9d35-2442dd2ec31b.png)

Sufiyan said that this is in fact darkest primary instead of having  Alpha